frontend/src/main/java/puf/frisbee/frontend/view/WaitingView.java

```diff
@@ -0,0 +1,29 @@
+package puf.frisbee.frontend.view;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.input.MouseEvent;
+import puf.frisbee.frontend.viewmodel.WaitingViewModel;
+
+public class WaitingView {
+    @FXML
+    private Label startLabel;
+
+    @FXML
+    private Button startButton;
+
+    private WaitingViewModel waitingViewModel;
+
+    public void init(WaitingViewModel waitingViewModel) {
+        // TODO: use dependency injection later on
+        // also later we will have the ViewHandler as dependency, so we can load different views when clicking a button
+        // (e.g. after confirming game over)
+        this.waitingViewModel = waitingViewModel;
+    }
+
+    @FXML
+    private void handleStartButtonClicked(MouseEvent event) {
+        startLabel.setText("Warte auf zweites Monster, um das Spiel zu beginnen...");
```

Comment:
Hier https://github.com/TroelsMortensen/Battleship/blob/master/src/client/view/login/LoginController.java#L53 ist das mit dem View-Swap - das kann also direkt in dieser View passieren, wir brauchen für den Wartebildschirm wohl erstmal auch kein ViewModel 🤔 ?